finesse/gui/temp_control.py

```diff
@@ -100,15 +89,27 @@ def _create_figure(self) -> None:
         self._canvas.draw()
 
     def _toggle_axis_visibility(self, name: str) -> None:
-        """Shows or hides BB plots."""
+        """Shows or hides BB plots.
```

Comment:
The abbreviation BB is probably not obvious to someone who doesn't know the system well. Just spell it out.